cliquet/tests/testappviews.py

```diff
@@ -0,0 +1,11 @@
+from cliquet.resource import BaseResource, ResourceSchema, crud
+import colander
+
+
+class MushroomSchema(ResourceSchema):
+    name = colander.SchemaNode(colander.String())
+
+
+@crud()
+class Mushroom(BaseResource):
+    mapping = MushroomSchema()
```

Comment:
Couldn't we factorize with this ? https://github.com/mozilla-services/cliquet/blob/exposed-cors-headers/cliquet/tests/resource/test_views.py#L16-L22